src/redux/slices/courseDetailsSlice.ts

```diff
@@ -0,0 +1,62 @@
+import { createAsyncThunk, createSlice, PayloadAction  } from '@reduxjs/toolkit';
```

Comment:
Might be better idea to use double quotes as in all previous files.
Would you mind to check all imports here?